src/GraphControl/DirectX/RenderMain.cpp

```diff
@@ -323,11 +350,11 @@ namespace GraphControl::DX
                                     m_nearestPointRenderer.SetColor(D2D1::ColorF(dotColor.R * 65536 + dotColor.G * 256 + dotColor.B, 1.0));
                                 }
 
-                                m_TraceLocation = Point(nearestPointLocationX, nearestPointLocationY);
+                                m_TraceLocation = Point(outNearestPointLocationX, outNearestPointLocationY);
                                 m_nearestPointRenderer.Render(m_TraceLocation);
                                 m_Tracing = true;
-                                m_TraceLocation = Point(nearestPointLocationX, nearestPointLocationY);
-                                m_TraceValue = Point(nearestPointValueX, nearestPointValueY);
+                                m_TraceLocation = Point(outNearestPointLocationX, outNearestPointLocationY);
+                                m_TraceValue = Point(outNearestPointValueX, outNearestPointValueY);
```

Comment:
Not this PR, but it looks like there is a flaw here that is changing the result of the y value.

For the equation y=x*1.234567 when x=100200 the graphing engine correctly sets outNearestPointValueY = 123703.61340000000.

However, when we put this value into a Point, Point takes a float and converts this value to 123703.617

Reply:
Fixed, pushed updates.